code/controllers/subsystem/processing/orbits.dm

```diff
@@ -151,3 +160,89 @@ PROCESSING_SUBSYSTEM_DEF(orbits)
 	for(var/obj/machinery/computer/objective/computer as() in GLOB.objective_computers)
 		for(var/M in computer.viewing_mobs)
 			computer.update_static_data(M)
+
+//Saves custom shuttles
+/datum/controller/subsystem/processing/orbits/proc/save_custom_shuttles()
+
+	if(!CONFIG_GET(flag/save_shuttle_ruins))
+		message_admins("Saving shuttles skipped, it is disabled in the config!")
+		return
+
+	//Calculate saved shuttle filesize
+	var/shuttle_filepath = CONFIG_GET(string/shuttle_ruin_filepath)
+
+	//Save shuttle
+	message_admins("SSORBITS: Saving custom shuttle ruins...")
+
+	var/list/old_files = list()
+	var/current_size = 0
+
+	//Find the current filesize saved
+	if(fexists(shuttle_filepath))
+		var/list/files = flist(shuttle_filepath)
+		//Calculate length of files
+		for(var/f in files)
+			var/fullF = "[shuttle_filepath][f]"
+			var/filelength = length(file(fullF))
```

Comment:
Look into whether it's faster to do this or store the file size in the file name